src/states/node.rs

```diff
@@ -551,6 +551,8 @@ impl Node {
         Ok(())
     }
 
+    // This match construct is huge and rustfmt's rules don't yield good results
+    #[cfg_attr(rustfmt, rustfmt_skip)]
```

Comment:
Let's stick to rustfmt's formatting except in extreme cases (rustfmt bugs, formatted multi-line strings, or after a discussion with the whole team).